samza-core/src/main/java/org/apache/samza/config/TaskConfigJava.java

```diff
@@ -37,7 +37,7 @@
 public class TaskConfigJava extends MapConfig {
   // Task Configs
   private static final String TASK_SHUTDOWN_MS = "task.shutdown.ms";
-  public static final long DEFAULT_TASK_SHUTDOWN_MS = 5000L;
+  public static final long DEFAULT_TASK_SHUTDOWN_MS = 30000L;
```

Comment:
@navina
Instead of overriding defaults, wondering if it's acceptable to default `task.shutdown.ms` to `30s` if undefined in standAlone container launcher flow, there by not affecting yarn deployments.